src/videojs-http-streaming.js

```diff
@@ -600,10 +600,46 @@ if (!videojs.use) {
 videojs.m3u8 = m3u8;
 videojs.options.hls = videojs.options.hls || {};
 
+const getTimeRanges = (player, type) => {
+  const timeRangesList = [];
+  const timeRanges = player[type]();
+
+  for (let i = 0; i < timeRanges.length; i++) {
+    timeRangesList.push({
+      start: timeRanges.start(i),
+      end: timeRanges.end(i)
+    });
+  }
+
+  return timeRangesList;
+};
+
+const getPlaybackStats = function() {
+  const player = this;
+
+  return {
+    videoPlaybackQuality: player.getVideoPlaybackQuality(),
+    playerDimensions: player.currentDimensions(),
+    hlsStats: player.tech_.hls.stats,
```

Comment:
do we want to call this `hlsStats` since the stats apply to dash sources as well?

Reply:
This should probably be renamed `vhsStats` since it would be the `vhs.stats` object